pkg/cmd/cli/cmd/deploy.go

```diff
@@ -317,7 +322,18 @@ func (o *DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer
 		return fmt.Errorf("couldn't cancel deployment %s", strings.Join(failedCancellations, ", "))
 	}
 	if !anyCancelled {
-		fmt.Fprintln(out, "no active deployments to cancel")
+		if latest == nil {
+			// TODO: this could mean that a new deployment is forthcoming but hasn't
+			// yet been created; might not be worth trying to express in this
+			// context.
+			fmt.Fprintf(out, "no deployments are in progress\n")
+		} else {
+			timeAt := strings.ToLower(units.HumanDuration(time.Now().Sub(latest.CreationTimestamp.Time)))
```

Comment:
I would prefer moving [formatRelativeTime](https://github.com/openshift/origin/blob/392e9db042c41681f5bbc60e0d4c8febf32ed552/pkg/cmd/cli/describe/helpers.go#L113) probably under `pkg/util/time` or at least export it (we already import its package) and use that instead but you would have to change a couple of other places as well.